SwiftUiMovieDb/Configuration/Configuration.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  Configuration.swift
+//  SwiftUiMovieDb
+//
+//  Created by mac on 03/11/23.
+//
+
+import Foundation
```

Comment:
Add TODO Comments, what you will be doing in this file, if it's empty.

//TODO: Need to add Configuration